lib/src/parse/stylesheet.dart

```diff
@@ -1772,7 +1772,7 @@ relase. For details, see http://bit.ly/moz-document.
 
       if (!scanner.scanChar($comma)) {
         scanner.expectChar($rparen);
-        return first;
+        return new ParenthesizedExpression(first, scanner.spanFrom(start));
```

Comment:
It is nice that this is an exceedingly simple implementation. Like you say in the description, it only adds complexity when parens are used, and this is the only line in source code that instantiates new Parenthesized Expression nodes.